# ekf_config.yaml
length: 0.5
width: 0.4
wheel_radius: 0.1
initial_state: [-2.15, -6.906, 0, 0, 0, 0]


Q:
- [0.05, 0, 0, 0, 0, 0]
- [0, 0.05, 0, 0, 0, 0]
- [0, 0, 0.01, 0, 0, 0]
- [0, 0, 0, 0.001, 0, 0]
- [0, 0, 0, 0, 0.001, 0]
- [0, 0, 0, 0, 0, 0.0005]

P0:
  - [0.5, 0, 0, 0, 0, 0]
  - [0, 0.5, 0, 0, 0, 0]
  - [0, 0, 0.3, 0, 0, 0]
  - [0, 0, 0, 0.1, 0, 0]
  - [0, 0, 0, 0, 0.1, 0]
  - [0, 0, 0, 0, 0, 0.05]

R_imu:
  - [0.02, 0, 0]
  - [0, 0.02, 0]
  - [0, 0, 0.1]
R_encoder:
  - [0.005, 0, 0]
  - [0, 0.005, 0]
  - [0, 0, 0.0001]

